src/sass/champion_overwrite.scss

```diff
@@ -235,6 +235,10 @@ ul.checked.small > li {
     font-weight: bold;
 }
 
+.text-normal {
```

Comment:
I think you wanted to remove the class as well as it's not in use anymore.
Thanks.

Reply:
Done, somehow forgot I added this class as well. Thanks.

Reply:
Done, somehow forgot I added the class. Thanks